js/private/test/repositories_checked.bzl

```diff
@@ -182,15 +182,6 @@ def npm_repositories():
             "sourcemap-codec": ["1.4.8"],
             "@types/estree": ["0.0.39", "0.0.51"],
             "fs.realpath": ["1.0.0"],
-            "inflight": ["1.0.6"],
-            "inherits": ["2.0.4"],
-            "minimatch": ["3.1.2"],
-            "once": ["1.4.0"],
-            "path-is-absolute": ["1.0.1"],
-            "wrappy": ["1.0.2"],
-            "brace-expansion": ["1.1.11"],
-            "balanced-match": ["1.0.2"],
-            "concat-map": ["0.0.1"],
```

Comment:
It's not clear to me why using my yaml parser resulted in some changes to this file.